libpod/lock/shm/shm_lock_test.go

```diff
@@ -27,6 +27,10 @@ const lockPath = "/libpod_test"
 
 // We need a test main to ensure that the SHM is created before the tests run
 func TestMain(m *testing.M) {
+	_, err := os.Stat("/dev/shm" + lockPath)
+	if !os.IsNotExist(err) {
+		os.RemoveAll("/dev/shm" + lockPath)
```

Comment:
You can save yourself the `stat`, and simply just unconditionally run `RemoveAll()`. Easier to maintain that way.

Wow does it ever super-duper disturb me to see hardcoded `/dev/shm`. I just ratholed into `shm_open()` and friends, and see no better way, but yeesh.